### PR TITLE
Set up Firebase

### PR DESCRIPTION
**Pull request by AnyaLeo**
The app worked successfully with Cloud Firestore database (the test is **not** included in this pull request)
The `google-services.json` file is not included in the repo and will be shared privately (it's not really clear if that can or cannot be committed to a public repo)

### CODE REVIEW

**Review by cl418 — APPROVED** [review]

**Comment by kachen10** [comment]
> The app worked successfully with Cloud Firestore database (the test is **not** included in this pull request)
> 
> The `google-services.json` file is not included in the repo and will be shared privately (it's not really clear if that can or cannot be committed to a public repo)

Good move. We should not push google services json to a public repo.

File: serious_stuff/pubspec.yaml [hunk 1]
```diff
@@ -23,6 +23,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  firebase_core: ^0.4.0+9
+
+  # Dependencies for the Firebase products
```
Comment (kachen10) [inline serious_stuff/pubspec.yaml:28]: Should this comment be moved above  line 26, above the firebase core dependency?

**Review by kachen10 — APPROVED** [review]
LGTM

File: serious_stuff/pubspec.yaml (hunk 1 above)
Comment (AnyaLeo) [reply serious_stuff/pubspec.yaml:28]: Oh, good point, will do that

**Pull request merged**